re/aes.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
+
+package re
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"encoding/hex"
+	"fmt"
+	"log"
+	"strings"
+)
+
+func decrypt(keyHex string, iv string, encryptedHex string) string {
+	/*
+	AES CBC-128 DECRYPTION requires 3 data fields
+	1. Key (16 bytes)
+	2. Initialization Vector (16 bytes)
+	3. Encrypted Data (16 bytes or length be multiple a of 16)
+
+	The encrypted data is divided into blocks of 16 bytes (128 bits) which then operated on with the IV and Key.   
+	*/
+	
+	//convert key hex string to bytes

```

Comment:
Put space after `//`, be consistent and start all comments with a capital letter.